great_expectations/data_context/data_context/data_context.py

```diff
@@ -279,6 +280,34 @@ def __init__(
         if self._check_for_usage_stats_sync(project_config):
             self._save_project_config()
 
+    @staticmethod
+    def _resolve_cloud_args(
+        cloud_mode: Optional[bool] = False,
+        cloud_base_url: Optional[str] = None,
+        cloud_access_token: Optional[str] = None,
+        cloud_organization_id: Optional[str] = None,
+        # <GX_RENAME> Deprecated as of 0.15.37
+        ge_cloud_mode: Optional[bool] = False,
+        ge_cloud_base_url: Optional[str] = None,
+        ge_cloud_access_token: Optional[str] = None,
+        ge_cloud_organization_id: Optional[str] = None,
+    ) -> Tuple[Optional[str], Optional[str], Optional[str], bool]:
```

Comment:
Can we make this a `NameTuple`?

Reply:
maybe later but this is for a hotfix - going to prioritize getting it if that's alright